pkg/asset-manager-utils/test/RewardsAssetManager.test.ts

```diff
@@ -105,7 +106,9 @@ describe('Rewards Asset manager', function () {
         upperCriticalPercentage: 4,
         lowerCriticalPercentage: 2,
       };
-      await assetManager.connect(poolController).setConfig(poolId, encodeInvestmentConfig(updatedConfig));
+      await pool.instance
+        .connect(poolController)
```

Comment:
This seems wrong, in line 100 `poolController` is assigned to `lp`?

Reply:
I've removed this as `MockAssetManagedPool` allows anyone to update the config so this is unnecessary.